securedrop/tests/functional/functional_test.py

```diff
@@ -131,3 +146,20 @@ def wait_for(self, function_with_assertion, timeout=5):
                 time.sleep(0.1)
         # one more try, which will raise any errors if they are outstanding
         return function_with_assertion()
+
+    def _alert_wait(self):
+        WebDriverWait(self.driver, 5).until(
+            expected_conditions.alert_is_present(),
+            'Timed out waiting confirmation popup to appear.')
```

Comment:
Nit: `waiting confirmation popup'` -> `waiting for confirmation popup`

(similarly below)